catalyst_voices/lib/widgets/toggles/voices_switch.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:flutter/material.dart';
+
+/// A Voices version of a [Switch] widget with optional thumb icon
+/// customization.
+///
+/// This widget provides a basic switch functionality with the ability to
+/// display a custom icon on the thumb.
+///
+/// {@category Widgets}
```

Comment:
I think you decided to remove it in other place, if so then lets remove it from here too